src/Controllers/AddTaskController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace App\Controllers;
+use App\Models\TasksModel;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+
+/**
+ * Class AddTaskController
+ * @package App\Controllers
+ * @return Response with header
+ */
+class AddTaskController {
+    protected TasksModel $model;
+    /**
+     * AddTaskController constructor.
+     * @param TasksModel $model
+     */
+    public function __construct (TasksModel $model)
+    {
+        $this->model = $model;
+    }
+    /**
+     * @param Request $request
+     * @param Response $response
+     * @param $args
+     * @return Response
+     */
+    public function __invoke(Request $request, Response $response, $args): Response
+    {
+        $data = $request->getParsedBody();
+        $newTask = $data['task_name'];
+        if ($newTask != "") {
+            $newTask = filter_var($newTask, FILTER_SANITIZE_STRING);
```

Comment:
Once checking isset I would then just pass filter_var `$data['task_name']` directly rather than first saving it in a variable.